pkg/api/annotations.go

```diff
@@ -30,17 +33,32 @@ func GetAnnotations(c *middleware.Context) Response {
 	result := make([]dtos.Annotation, 0)
 
 	for _, item := range items {
+		// Get user info for annotation event
+		userName := ""
+		if item.UserId != 0 {
+			userQuery := &models.GetUserByIdQuery{
```

Comment:
I figured as much, will try to figure out how we should do this. need a read model that is different then the write model